profiles/common/modules/custom/tmgmt_workbench/tmgmt_workbench.module

```diff
@@ -113,66 +116,37 @@ function tmgmt_workbench_form_tmgmt_entity_ui_translate_form_alter(&$form, $form
   if ($entity_type == 'node') {
     $node = $form_state['entity'];
     if (workbench_moderation_node_moderated($node)) {
+      // If the tmgmt_poetry module is disabled remove possibility to select
+      // multiple languages and by doing that follow the default tmgmt path.
```

Comment:
Here if tmgmt_poetry is not active we are actually changing the default tmgmt behavior, which is to set $form['languages']['#multiple'] = TRUE;

So the comment is misleading.